indexer/feature_data.cpp

```diff
@@ -421,29 +430,37 @@ void FeatureParams::SetRwSubwayType(char const * cityName)
   }
 }
 
-bool FeatureParams::FinishAddingTypes()
+FeatureParams::TypesResult FeatureParams::FinishAddingTypesEx()
 {
   base::SortUnique(m_types);
 
+  TypesResult res = TYPES_GOOD;
+
   if (m_types.size() > kMaxTypesCount)
   {
     UselessTypesChecker::Instance().SortUselessToEnd(m_types);
 
-    LOG(LWARNING, ("Exceeded max types count, got total", m_types.size(), ":", TypesToString(m_types)));
+    LOG(LWARNING, ("Exceeded max types count:", TypesToString(m_types)));
```

Comment:
Did you overwrite it by accident? It was changed in d01929a3ee973c38121446a1e73646aee115fb02

I'm worried some other more important changes could be reverted elsewhere..

Reply:
Should be deleted at all, seems like a mistake in a merge conflict.

Reply:
But total types count was very useful, please add to the new log line.